app/scenes/Settings/People.js

```diff
@@ -87,27 +93,31 @@ class People extends React.Component<Props> {
           Invite people…
         </Button>
 
-        <Tabs>
+        <TotalCount $loading={!count.all}>
+          {count.all} total {count.all > 1 ? "users" : "user"}
+        </TotalCount>
```

Comment:
Lets remove this for now, it isn't super useful called out and is included on the "Everyone" tab below